Assets/HoloToolkit/UX/Scripts/Keyboard.cs

```diff
@@ -218,10 +234,48 @@ protected override void Awake()
             RectTransform rect = GetComponent<RectTransform>();
             m_ObjectBounds = new Vector3(canvasBounds.size.x * rect.localScale.x, canvasBounds.size.y * rect.localScale.y, canvasBounds.size.z * rect.localScale.z);
 
+            // Actually find microphone key in the keyboard
+            var dicationButton = RecursiveFindChild(gameObject.transform, "Dictation");
+            if (dicationButton != null)
+            {
+                var dicationIcon = dicationButton.transform.Find("keyboard_closeIcon");
+                if (dicationButton != null)
+                {
+                    _recordImage = dicationButton.GetComponentsInChildren<Image>()[1];
+                    var material = new Material(_recordImage.material);
+                    _defaultColor = material.color;
+                    _recordImage.material = material;
+                }
+            }
+
+
             // Keep keyboard deactivated until needed
             gameObject.SetActive(false);
         }
 
+        /// <summary>
+        /// Recursive find. Potentially move to a child class.
+        /// </summary>
+        GameObject RecursiveFindChild(Transform parent, string childName)
```

Comment:
This could be a GameObject Extension method.  I wonder if something similar already exists?
I'll take a look.

Reply:
You are right. There is a Utils.GetChildRecursive. Apparently I did not search well enough when I added this. Updated the PR to use that method and removed mine from Keyboard.cs

Reply:
Looks like this was removed in https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/1496/commits/9fc3bade2abdc74c119d72240bff10f86fbf3ff9 but re-added in https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/1496/commits/b4e89803adf6b46c6770a8df869e5e590dc262d2. That second commit seems to have reverted both the first commit I linked and https://github.com/Microsoft/MixedRealityToolkit-Unity/pull/1496/commits/0c6e01c1ebc877f93ea52585318ae6b0c02e777b. Was that intended?